### PR TITLE
Merging code from Main

### PR DESCRIPTION
**Pull request by BrianYoon91**
-added header, displayText, Form, and Footer components. 
-installed axios and created a successful call to make a call with user selection
-2 user interaction is used to make an axios call, category selection and number selection. 
-functions passed down as props to retrieve user info.

### CODE REVIEW

File: src/Form.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+
+const Form = (props) => {
+
+    // State for user selection for category and number
+
+    const [userValue, setUserValue] = useState(""); 
+
+    const [numberSelected, setNumberSelected] = useState("");
```
Comment (Nickduke) [inline src/Form.js:9]: Maybe you could have passed state down from app instead of adding new state within the Form component?

**Review by Nickduke — COMMENTED** [review]
Project and code both look great! Consider sharing some state within your components if possible.

File: src/DisplayExcuses.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                ): (
+
+                    <div className="excusesContainer">
+                        {
+                            props.excuses.map((lie)=>{
+                                return (
+                                    
+                                    <div className="excuses" key={lie.id}>
+                                        <h3>{lie.excuse}</h3>
+                                    </div>
```
Comment (genc1499) [inline src/DisplayExcuses.js:17]: Semantically, I would consider putting each lie in a list.

**Review by genc1499 — COMMENTED** [review]
Awesome job Brian, design is nice and clean and the app works great!

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,234 @@
+@import url('https://fonts.googleapis.com/css2?family=Bowlby+One+SC&display=swap');
```
Comment (Nickkuemmling) [inline src/App.css:9]: Don't forget to add your setup snippet.

**Review by Nickkuemmling — COMMENTED** [review]
Looking great! I love the use of the emoji's to bring the page to life and the Harry Potter reference. Just don't forget to add the setup snippet in your App.css.

**Pull request merged**